connect_your_app_to_IRIS/technical_details/eps-config/settings/staging/certs/staging-root.crt

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Should this be checked in?

Reply:
Ah I think this is the cert that you trust on Staging environment right?

Reply:
Yes, that's the trusted cert for staging. Can therefore be stored in the respository.